ddtrace/appsec/_processor.py

```diff
@@ -434,3 +436,10 @@ def on_span_finish(self, span: Span) -> None:
                     del self._span_to_waf_ctx[s]
                 except Exception:  # nosec B110
                     pass
+
+
+# load waf at the end only to avoid possible circular imports with gevent
+import ddtrace.appsec._ddwaf as ddwaf  # noqa: E402
+from ddtrace.appsec._metrics import _set_waf_init_metric  # noqa: E402
+from ddtrace.appsec._metrics import _set_waf_request_metrics  # noqa: E402
+from ddtrace.appsec._metrics import _set_waf_updates_metric  # noqa: E402
```

Comment:
For these imports that are not used here, maybe worth just importing them directly wherever they are used instead of using this class as a proxy?

Reply:
As they are used inside the AppsecSpanProcessor where everything is critical, I prefer not to import them locally, that's why I kept the import global, but at the end to avoid circular errors.